diaphora.py

```diff
@@ -2068,13 +2071,19 @@ def diff(self, db):
       cur.close()
     return True
 
+g_debug = False
```

Comment:
Please don't get to invent your own debugging method, call: `debug_refresh()`.

Reply:
OK will do